packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -6,7 +6,8 @@ import type { ExploreFilter, ExploreQuery, TimeRangeV4 } from '@kong-ui-public/a
 export interface DashboardRendererContext {
   filters: ExploreFilter[]
   timeSpec: TimeRangeV4
-  tz?: string
+  tz?: string,
+  refreshInterval?: number
```

Comment:
I don't feel like this is a good fit for context.  It seems like it should be part of the definition.  Maybe we can discuss after all the meetings?

Reply:
I just wanted to put it in a spot that made the least amount of code changes, and not add additional props to the component 

I thought the context is a perfect spot for it, makes sense in my mind as it applies to the whole dashboard.

Or are you thinking we should specify refresh interval per tile?

Reply:
(Resolved out of band -- I retract my objection, sorry about that.)